commands.json

```diff
@@ -4879,7 +4895,7 @@
     },
     "GEORADIUSBYMEMBER_RO": {
         "summary": "A read-only variant for GEORADIUSBYMEMBER",
-        "since": "3.2.10",
+        "since": "5.0.0",
```

Comment:
need to check this: https://github.com/redis/redis/pull/10034

Reply:
@enjoy-binbin thanks for catching.
apparently @itamarhaber modified (and merged) the contents of commands.json in redis-doc directly, and forgot to merge the change in redis itself.
so it looks as if this PR reverts that chagne.
i now merged https://github.com/redis/redis/pull/10034 so this change will be fixed sooner or later.